operator-framework-core/src/main/java/io/javaoperatorsdk/operator/api/config/LeaderElectionConfiguration.java

```diff
@@ -70,7 +72,18 @@ public Duration getRetryPeriod() {
     return retryPeriod;
   }
 
-  public Optional<String> getIdentity() {
-    return Optional.ofNullable(identity);
+  public String getIdentity() {
```

Comment:
TBH I don't think this is good, would like to keep the POJOs clear without any logic